scripts/ci-build.sh

```diff
@@ -296,7 +296,10 @@ stage_artifact mv gdp-src-build/tmp/deploy/licenses/genivi-dev-platform*/license
 stage_artifact mv gdp-src-build/tmp/deploy/sdk*
 stage_artifact cp gdp-src-build/tmp/deploy/images/*
 stage_artifact cp gdp-src-build/conf/*.conf
-stage_artifact cp logs.tar.gz
+stage_artifact mv logs.tar.gz
+stage_artifact cp gdp-src-build/tmp/buildhistory/images/*/glibc/genivi-dev-platform/files-in-image.txt
+stage_artifact mv gdp-src-build/tmp/buildhistory
```

Comment:
Yes, that is correct.  There will be two copies of files-in-image.txt in the result.  But the file is small and I wanted it to be easy to find at the top of the staging directory, and with a well-known path.  To get to the other files, you would look in the full build history directory.